tools/upload_text.php

```diff
@@ -41,6 +41,15 @@
     and the upload does not succeed, upload a small placeholder zip file 
     instead and email %s for assistance."), $db_requests_email_addr);
 
+// Deny post_1 and return_1 if the project is currently in SR
+if (($stage == 'post_1' || $stage == 'return_1') &&
+    $project->is_available_for_smoothreading())
+{
+    $title = _("Disabled during Smooth Reading");
+    $body = _("This function is disabled while the project is in the Smooth Reading Pool.<br/><br/>If you believe this is an error, please contact db-req for assistance.");
```

Comment:
For translation purposes and HTML rendering, I would do:
```
$body = '<p>' . _("This function is disabled...") . '</p>' .
        '<p>' . _("If you believe this...") . '</p>';
```
Also note that we use HTML5, not XHTML, so it would be `<br>` not `<br/>` if you were to use them.